### DISCUSSION TITLE
Playing a video as a background in Web Page Wallpaper

### DISCUSSION THREAD

**Discussion by mastermach50** [rocksdanister/lively#1888] [Q&A] [answered]
I tried to play a video as a background in a web page wallpaper. It does work in browser (Firefox) but when imported to Lively the video does not load.

How can I fix this?

[This](https://github.com/MasterMach50/Inazuma-Lively-Wallpaper) is the project.

https://mathewma3.in/Inazuma-Lively-Wallpaper/
This is how the wallpaper should look.

**Comment by rocksdanister** [comment, marked as answer]
This format is not supported due to licensing, convert the video to VP9:
https://github.com/rocksdanister/lively/wiki/Web-Guide-VI-:-Webpage-Video-Support

For further customization you can refer:
https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data

**Reply by mastermach50** [reply]
Thanks, it now works.

> I converted it to av1 since vp9 encoding was too slow on ffmpeg.
